packages/manager/modules/manager-components/src/resource-selector/index.scss

```diff
@@ -0,0 +1,14 @@
+ovh-manager-resource-selector {
+  @import '~bootstrap4/scss/_functions.scss';
+  @import '~bootstrap4/scss/_variables.scss';
+  @import '~bootstrap4/scss/_mixins.scss';
+  @import '~bootstrap4/scss/_utilities.scss';
+  @import '~bootstrap4/scss/_grid.scss';
```

Comment:
Keep in mind that there's a work to remove less tilde imports (see: https://github.com/ovh/manager/pull/5015).

Reply:
Yeah... But I think the scss imports have to be handled separately...
#5015 handles only `.less` imports

Reply:
my bad, yes it does not concern scss imports

Reply:
Hello, the PR concerning `~` on sass import was merged : https://github.com/ovh/manager/pull/4994

Reply:
This has been corrected. Thanks